modules/relocation-of-openshift-jenkins-images.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: CONCEPT
+
+[id="relocation-of-openshift-jenkins-images_{context}"]
+= Relocation of OpenShift Jenkins images
+
+.What stays the same with the OpenShift Jenkins images?
+
+* The Cluster Samples Operator manages the `ImageStream` object and `Template` objects for operating the OpenShift Jenkins images.
```

Comment:
The Cluster Samples Operator manages the `ImageStream` object*s*

use plural for both

Reply:
Updated. Thanks.